src/UpdatedDate.js

```diff
@@ -10,19 +10,36 @@ export default function UpdatedDate(props) {
     "Friday",
     "Saturday",
   ];
-  let day = days[props.date.getDay()];
-  let hours = props.date.getHours();
-  let minutes = props.date.getMinutes();
+
+  props = props.date;
+
+  let correctedOffset = ((props.timeZoneOffset * 100) / 60 / 60);
```

Comment:
How did you determine this math expression? Google?

Reply:
let props.timeZoneOffset = the timezone offset denoted in seconds
let offsetInMinutes = props.timeZoneOffset / 60
let offsetInHours = props.offsetInMinutes / 60

The `* 100` on ln 16 and the `/ 100` on ln 17 are not needed.
This is left over (and forgotten) from when I initially set out to understand what units the offset represented.
Eventually I either figured it out or ended up here https://openweathermap.org/current#current_JSON.

Either way, I forgot to remove the *100 and /100.